### ISSUE TITLE
validate package scoping causes incorrect package namespacing

### ISSUE THREAD

**Issue by majuscule** [DiffSK/configobj#128]
I believe that I have found a bug in the `configobj`/`validate` interaction. I am not entirely sure if this code is causing a user-facing issue, but I am fairly certain that the code below is not behaving as intended.

https://github.com/DiffSK/configobj/blob/master/src/configobj/__init__.py#L2177
excerpt:

``` python
if not preserve_errors or isinstance(cause, self._vdtMissingValue):
```

The import of validate ([__init__.py#L2142](https://github.com/DiffSK/configobj/blob/master/src/configobj/__init__.py#L2142)) is relative, and the resulting classes are namespaced within configobj: `<class 'configobj.validate.VdtMissingValue'>` - causing the above test to fallthrough.

This is causing additional problems when I attempt to raise `VdtMissingDefinition` (PR
https://github.com/DiffSK/configobj/pull/127) from configobj, as the resulting `preserve_errors` dictionary contains exceptions with differing namespaces, forcing awkward imports from external code which would like to test entry error types.

**Comment by jun66j5** [comment]
`_version.py` has the same problem, violates packaging namespace. It should be `configobj/_version.py`.

**Comment by robdennis** [comment]
I think this is addressed in 5.0.7, but since I needed to manually implement the fix, please reopen if I messed it up
